playwright/tests/homepage.spec.ts

```diff
@@ -2,7 +2,23 @@ import { test, expect } from "@playwright/test";
 import { DCHomepage } from "../pages/homepage.page";
 
 test.beforeEach(async ({ page }) => {
-  await page.goto("/", { waitUntil: "domcontentloaded" });
+  // Block analytics, tracking, and third-party domains
+  await page.route(/.*googletagmanager\.com.*/, (route) => route.abort());
+  await page.route(/.*demdex\.net.*/, (route) => route.abort());
+  await page.route(/.*adobedtm\.com.*/, (route) => route.abort());
+  await page.route(/.*everesttech\.net.*/, (route) => route.abort());
+  await page.route(/.*ipify\.org.*/, (route) => route.abort());
+  await page.route(/.*google\.com.*/, (route) => route.abort());
+  await page.route(/.*omappapi\.com.*/, (route) => route.abort());
+  await page.route(/.*google-analytics\.com.*/, (route) => route.abort());
```

Comment:
nice! if we find this helpful on the homepage, we could make a change to abort these calls for all tests on all pages using a global setup file. definitely doesn't need to happen on this PR.